src/coreclr/jit/compiler.h

```diff
@@ -3809,7 +3810,8 @@ class Compiler
     bool gtStoreDefinesField(
         LclVarDsc* fieldVarDsc, ssize_t offset, unsigned size, ssize_t* pFieldStoreOffset, unsigned* pFieldStoreSize);
 
-    void gtPeelOffsets(GenTree** addr, target_ssize_t* offset, FieldSeq** fldSeq = nullptr);
+    void gtPeelOffsets(GenTree** addr, target_ssize_t* offset, FieldSeq** fldSeq = nullptr) const;
+    static void gtPeelOffsetsI32(GenTree** addr, int* offset);
```

Comment:
Looks unused?

Reply:
Yep, an artifact from the previous impl when the phase was after Loop Clonning.